ember-app/app/templates/components/hb-filter-group.hbs

```diff
@@ -1,5 +1,10 @@
 <h5 class='filter-header'>
   {{name}}
+  {{#if activeFilters.length}}
+    <small>({{activeFilters.length}})</small>
+  {{/if}}
   <i class="ui-icon {{if isCollapsed 'ui-icon-triangle-1-e' 'ui-icon-triangle-1-s'}}"></i>
 </h5>
-{{yield}}
+{{#each filters as |filter|}}
+  {{hb-filter name=filter.name mode=filter.mode color=filter.color tagType="colored" queryParam="label"}}
```

Comment:
The `tagType` I did omit on accident. `queryParam` is deprecated for that component, just never got taken out.